### PR TITLE
add minutes for most recent meeting

### PR DESCRIPTION
**Pull request by Trott**

### CODE REVIEW

File: meetings/2017-12-13.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+   * N-API - continue to work on moving out of ‘experimental’.  Looking to use bcrypt and
+     one of Anna’s modules for production use.  Have made some progress on investigating
+     how node-pre-gyp will work, focus on tests and doc continues.
+
+  * Moderation: Meeting today. Will have a “real” update next time.
+
+  * V8: Opened PR for 6.4. Not expecting major issues with this update.
+
+  * Module: There was the blog post on the
```
Comment (fhinkel) [inline meetings/2017-12-13.md]: Add:

 Node.js Foundation's Medium Publication [The Current State of Implementation and Planning for ESModules](https://medium.com/the-node-js-collection/the-current-state-of-implementation-and-planning-for-esmodules-a4ecb2aac07a)

File: meetings/2017-12-13.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+“Yes” votes from this meeting’s attendes and email: Rich, Franzi, Michaël Zasso, Trevor, Nikita, Anna, James, Michael Dawson, Colin, Evan, Matteo
+
+[Post-meeting note from Rich: That’s enough votes to declare it official. Will move forward with it on GitHub.]
+
+* Document guidelines for accountability expectations for all members of the TSC and CommComm [#311](https://github.com/nodejs/TSC/issues/311)
+
+Franzi: Move to next week.
+
+Rich: Yes, agreed. Let’s gather votes for people who are here, though.
```
Comment (fhinkel) [inline meetings/2017-12-13.md]: I think this is for the above issue?

File: meetings/2017-12-13.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+### Announcements
+* Collab summit Berlin, most likely on May 31 - Friday, June 1, 2018. The 2 days prior to JSConf.eu [#56](https://github.com/nodejs/summit/issues/56).
+ 
+*Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* events: add off alias to removeListener [#17156](https://github.com/nodejs/node/pull/17156)
+
+Rich: GitHub conversation did not get “unstuck”. Move to vote.
```
Comment (ljharb) [inline meetings/2017-12-13.md]: What was the outcome here? Was a vote taken? If not, when will it be taken?

File: meetings/2017-12-13.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+Franzi: Move to next week.
+
+Rich: Yes, agreed. Let’s gather votes for people who are here, though.
+
+* unclear membership related terms and processes [#178](https://github.com/nodejs/TSC/issues/178)
+
+Franzi: Myles put it on the agenda and wanted to know if we should work on it or close it. I vote to close it since we can always re-open or open a more targeted issue.
+
+RIch: +1
```
Comment (fhinkel) [inline meetings/2017-12-13.md]: s/RIch/Rich

File: meetings/2017-12-13.md (hunk 1 above)
Comment (ljharb) [inline meetings/2017-12-13.md]: What’s the rest of this sentence?

File: meetings/2017-12-13.md (hunk 3 above)
Comment (fhinkel) [reply meetings/2017-12-13.md]: IIRC we want for the TSC members to vote in the issue. We might already have enough approvals there, let me check.

Comment (fhinkel) [reply meetings/2017-12-13.md]: Not enough votes yet. Left a comment.

File: meetings/2017-12-13.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+Rich: GitHub conversation did not get “unstuck”. Move to vote.
+
+
+* DO NOT MERGE: \[module\] Revert checking for package.json before extensions [#15015](https://github.com/nodejs/node/pull/15015)
+
+Michaël: Need an update from Bradley
+
+Franzi: Leave it for next week’s meeting.
+
+Rich: Invite Bradley as a guess.
```
Comment (maclover7) [inline meetings/2017-12-13.md]: `guess` --> `guest`?

File: meetings/2017-12-13.md (hunk 3 above)
Comment (MylesBorins) [reply meetings/2017-12-13.md]: https://github.com/nodejs/node/pull/17156#issuecomment-351457062

File: meetings/2017-12-13.md (hunk 5 above)
Comment (MylesBorins) [reply meetings/2017-12-13.md]: /cc @mhdawson to make sure we don't forget this

File: meetings/2017-12-13.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+* build: update minimum kernel version to 3.10.0 [#14795](https://github.com/nodejs/node/pull/14795)
+
+Rich: Given this and the last issue, maybe we should invite Ruben to the meeting as an observer too next time to frame these issues.
```
Comment (MylesBorins) [inline meetings/2017-12-13.md]: /cc @mhdawson to remember to invite ruben

File: meetings/2017-12-13.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+Franzi: Agreed. We need more people to decide and frame these. Maybe we should invite Gibson too.
```
Comment (MylesBorins) [inline meetings/2017-12-13.md]: also + gibson

File: meetings/2017-12-13.md (hunk 5 above)
Comment (mhdawson) [reply meetings/2017-12-13.md]: @Trott volunteered to chair next week's meeting as I'm off on holiday until Jan 2 starting Friday.  The issue will be created with observers, @Trott can you add him once the issue is generated (~9AM EST)

File: meetings/2017-12-13.md (hunk 6 above)
Comment (mhdawson) [reply meetings/2017-12-13.md]: @Trott same comment as last time if you can add Ruben to next week's meeting.

File: meetings/2017-12-13.md (hunk 7 above)
Comment (mhdawson) [reply meetings/2017-12-13.md]: @Trott ditto

File: meetings/2017-12-13.md (hunk 5 above)
Comment (Trott) [reply meetings/2017-12-13.md]: Yeah, I made a note in my calendar for Monday to invite Bradley and Gibson as guests. I'll no doubt scan over the agenda to see if there's anyone else to invite and probably ask William Kapke too. I'll also double-check to make sure @danbev has been added to the list of attendees. I'm guessing that is something you might have to update in your cron script or whatever it is, @mhdawson.

**Pull request merged**